core/experiments/wp-cli.php

```diff
@@ -5,191 +5,223 @@
 use Elementor\Core\Experiments\Manager as Experiments_Manager;
```

Comment:
@Barcov You might still be busy. But there is a huge diff in this file that seems unnecessary.